console/console-init/ui/src/modules/address-detail/components/AddressLinksToolbar/AddressLinksToolbar.tsx

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+import {
+  DataToolbarItem,
+  DataToolbar,
+  DataToolbarContent,
+  SelectOptionObject,
+  DataToolbarChipGroup,
+  DataToolbarChip,
+  DataToolbarContentProps
+} from "@patternfly/react-core";
+import { ISortBy } from "@patternfly/react-table";
+import { SortForMobileView, useWindowDimensions } from "components";
+import { AddressLinksToggleGroup } from "../AddressLinksToggleGroup";
```

Comment:
as we are not using relative path . can you replace relative path with absolute path as we are following.